src/core/1/api/api-reference/controller-collection/create/index.md

```diff
@@ -74,16 +86,18 @@ Body:
 
 ## Arguments
 
-- `collection`: data collection to create
-- `index`: data index that will host the new data collection
+- `collection`: name of the collection to create
+- `index`: index name that will host the new collection
```

Comment:
Some times there is : 
- `index`: index name
And here : 
- `index`: index name that will host the new collection

I think we need to unify that